build-scripts/repo-conf/aos-signed-building.repo

```diff
@@ -101,6 +101,12 @@ baseurl=http://download.lab.bos.redhat.com/rcm-guest/puddles/RHAOS/AtomicOpenShi
 gpgcheck=0
 enabled=0
 
+[rhel-7-server-ose-3.8-rpms]
+name=rhel-7-server-ose-3.8-rpms
+baseurl=http://download.lab.bos.redhat.com/rcm-guest/puddles/RHAOS/AtomicOpenShift-signed/3.8/building/RH7-RHAOS-3.8/x86_64/os
+gpgcheck=1
```

Comment:
Looks like this is not like the others.

Reply:
Weird... no idea how I made that gpgcheck a 1 instead of 0... was all copy/paste. Fixed now.